src/main/java/mat/client/cqlworkspace/attributes/InsertFrhiAttributesDialogDisplay.java

```diff
@@ -0,0 +1,5 @@
+package mat.client.cqlworkspace.attributes;
+
+public interface InsertFrhiAttributesDialogDisplay {
```

Comment:
typo I guess Insert**Frhi**AttributesDialogDisplay :)

Reply:
Good catch! thank you!